lib/client/api.go

```diff
@@ -1746,7 +1748,7 @@ func (tc *TeleportClient) applyProxySettings(proxySettings ProxySettings) error
 		if err != nil {
 			return trace.BadParameter(
 				"failed to parse value received from the server: %q, contact your administrator for help",
-				proxySettings.SSH.ListenAddr)
+				proxySettings.SSH.SSHPublicAddr)
```

Comment:
This is the actual error that was fixed, everything else is just go-lint complaining